doc/nrf/ug_bootloader.rst

```diff
@@ -92,7 +92,96 @@ These signed variants can be used to perform an upgrade procedure through the :r
 Adding a bootloader chain to your application
 *********************************************
 
-See :ref:`bootloader_build_and_run` in the documentation of the :ref:`bootloader` sample for instructions on how to add an immutable bootloader as first stage of the secure bootloader chain.
+The |NCS| includes a sample implementation of an immutable bootloader.
+As upgradable bootloader, it provides and extends :doc:`mcuboot:index`.
 
-See :doc:`mcuboot:index` for information on how to implement an upgradable bootlader.
+Both bootloaders can easily be included in your application using :ref:`ug_multi_image`.
+
+Adding an immutable bootloader
+==============================
+
+To add an immutable bootloader to your application, set :option:`CONFIG_SECURE_BOOT` and add your private key file under :option:`CONFIG_SB_SIGNING_PUBLIC_KEY`.
+|how_to_configure|
+
+See the documentation of the :ref:`bootloader` sample for more information.
+The :ref:`bootloader_build_and_run` section has detailed instructions for adding the immutable bootloader as first stage of the secure bootloader chain.
+
+Adding an upgradable bootloader
+===============================
+
+To add an upgradable bootloader to your applications, set :option:`CONFIG_BOOTLOADER_MCUBOOT`.
+|how_to_configure|
+
+To ensure the root of trust, you should always include the immutable bootloader as well when adding an upgradable bootloader.
+
+.. note::
+   It is possible to include this bootloader without the immutable bootloader.
+   However, in this case the upgradable bootloader cannot actually be upgraded, because the root of trust is missing.
+
+   In the default configuration, most |NCS| samples include the upgradable bootloader but not the immutable bootloader.
+   The reason for this is that the immutable bootloader must be configured with a private key before use.
```

Comment:
AFAIK only a handful of samples includes mcuboot (upgardeable bootloader) - http_application_update, aws_fota, smp_svr, any more?

```suggestion
```

I don't see the value in these sentences, and to me they both look incorrect. The reason for not using the immutable bootloader is that it occupies space, and you might not need to be able to upgrade your bootloader.

Reply:
@ru-fu do you agree?

Reply:
+1 to @hakonfam's suggestion.

Reply:
Sure, we can remove the sentences if they are not needed.
The input I got was that to include the upgradable bootloader, you need to enable CONFIG_BOOTLOADER_MCUBOOT=y  and CONFIG_SECURE_BOOTLOADER=y. And I was confused why this is not the case in the samples we provide.